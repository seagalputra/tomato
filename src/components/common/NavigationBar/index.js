import React from 'react'
import tomatoIcon from 'assets/img/icon.svg'

const NavigationBar = () => {
  return (
    <header>
      <nav className="flex items-center justify-between flex-wrap bg-transparent p-6 border-b">
        <div className="flex items-center flex-shrink-0 mr-6">
          <img
            className="fill-current h-8 w-8 mr-3"
            src={tomatoIcon}
            width="54"
            alt="Tomato Icon"
          />
          <span className="font-bold text-xl tracking-tight">Tomato</span>
        </div>
      </nav>
    </header>
  )
}

export default NavigationBar
