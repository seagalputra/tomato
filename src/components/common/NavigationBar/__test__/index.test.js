import React from 'react'
import { shallow } from 'enzyme'
import NavigationBar from '..'

describe('NavigationBar', () => {
  it('renders without crashing', () => {
    const component = shallow(<NavigationBar />)
    expect(component).toMatchSnapshot()
  })
})
