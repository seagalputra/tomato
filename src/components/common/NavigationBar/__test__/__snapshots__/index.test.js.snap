// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBar renders without crashing 1`] = `
<header>
  <nav
    className="flex items-center justify-between flex-wrap bg-transparent p-6 border-b"
  >
    <div
      className="flex items-center flex-shrink-0 mr-6"
    >
      <img
        alt="Tomato Icon"
        className="fill-current h-8 w-8 mr-3"
        src="icon.svg"
        width="54"
      />
      <span
        className="font-bold text-xl tracking-tight"
      >
        Tomato
      </span>
    </div>
  </nav>
</header>
`;
